### ISSUE TITLE
Xcode 11.4 Swift Package Manager can't import ObjectMapper

### ISSUE THREAD

**Issue by mymbrooks** [tristanhimmelman/ObjectMapper#1082]
Before Xcode 11.4 Swift Package Manager can import ObjectMapper successful.
After update to Xcode 11.4 the error output:
target 'ObjectMapper' referenced in product 'ObjectMapper' is empty

I found in Xcode 11.4 release note

Swift Packages
New Features
Remote Swift packages with tools version 5.2 and above no longer resolve package dependencies that are only used in their test targets, improving performance and reducing the chance of dependency version conflicts. (56925017)
Swift Package Manager uses a new strategy for resolving package dependencies that significantly improves the quality of error messages and performance in complex package graphs. (45371461)

**Comment by ksloginov** [comment]
In beta 2, for me it's getting imported, however I can't build the project now:
![Screenshot 2020-02-26 at 15 28 29](https://user-images.githubusercontent.com/1670172/75354804-dc7d3100-58ad-11ea-98b0-ca07686bda9b.png)

I.e. compiler thinks that the files are duplicated. Then, looking at the file structure, I've noticed, that they are actually linked twice:
![Screenshot 2020-02-26 at 15 35 42](https://user-images.githubusercontent.com/1670172/75354709-bc4d7200-58ad-11ea-9c27-bf7688ea32d1.png)

![Screenshot 2020-02-26 at 15 32 41](https://user-images.githubusercontent.com/1670172/75355207-6e853980-58ae-11ea-9cd6-5c73ff40efd7.png)

It can't be just me.. Or?

**Comment by mymbrooks** [comment]
@ksloginov I update to Xcode 11.4 beta 2 the same error

**Comment by xernaj** [comment]
It might be ok in Xcode 11.4 beta 3.

**Comment by ksloginov** [comment]
@xernaj of course not, it's Apple after all.

**Current status**: fails in Xcode 11.4 beta 3.

**Comment by ksloginov** [comment]
@mymbrooks I've submitted a PR for this issue. With the forked version - it works for me now. Let me know if you're still experiencing the issues w/ PR'ed version

**Comment by zdnk** [comment]
Any update on this one? Xcode 11.4 is out.

**Comment by mymbrooks** [comment]
@ksloginov Sorry for late reply I update to 11.4 final, clean all cache, use your repos https://github.com/ksloginov/ObjectMapper.git  and still the same error.

**Comment by ksloginov** [comment]
@mymbrooks 
Huh.. Just tried with 11.4-release - still works fine for me (with mine repo).
![Screenshot 2020-03-26 at 12 37 19](https://user-images.githubusercontent.com/1670172/77642816-8f4cb780-6f5e-11ea-9d83-ba8a028c943b.png)
Thinking out loud - you're forking my master branch, right? (I was too lazy to create a separate release etc. for that)

Do you have any double-refs with my repo?
This is what I see in my real-project, which is depending on OM:

![Screenshot 2020-03-26 at 12 38 55](https://user-images.githubusercontent.com/1670172/77642981-dc308e00-6f5e-11ea-8b6b-a6d0947b8d50.png)

(Before, it used to be a cycle-reference here)

**Comment by mymbrooks** [comment]
@ksloginov yeah master branch and double refs, I will check source code carefully to find why

**Comment by mymbrooks** [comment]
@ksloginov It's my mistake, I clone 3.5.1 not master now it works fine.

**Comment by joshuapoq** [comment]
Definitely an issue with the ObjectMapper repo - you have folder infinite recursion through the random symbolic link in `Sources/ObjectMapper`. The compiler goes into recursion and fails.

Maybe it was a mistake symlink in a previous commit?

**Comment by ruizmarc** [comment]
Any chance to get the pull request merged?
